backend/mora/service/orgunit.py

```diff
@@ -105,7 +108,14 @@ def get(cls, c, type, objid):
             effect.get('tilstande')
                   .get('organisationenhedgyldighed')[0]
                   .get('gyldighed') == 'Aktiv'
-        ])
+        ]
+
```

Comment:
Her har vi igen en stor list comprehension direkte i return. Jeg aerkender at de findes i MO i forvejen, men det bliver aldrig noget jeg bliver fan af.

Reply:
Som tidligere nævnt er det ikke ny kode - det bliver forhåbentlig løst på et tidspunkt, men er out of scope for nu.